stable/k8s-secret-updater/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes SecretUpdater
+name: secret-updater
+version: 0.1.1
+maintainers:
+  - name: FFX Blue Operations
+    email: blueops@fairfaxmedia.com.au
```

Comment:
please update/correct this email address

Reply:
that's the same as the other charts in this repo...

Reply:
did we get github@fairfaxmedia.com.au set up?

Reply:
Edited!